sycl/test-e2e/Graph/Error/lit.local.cfg

```diff
@@ -1 +1,2 @@
 config.required_features += ['aspect-ext_oneapi_limited_graph']
+config.unsupported_features += ['gpu-intel-dg2']
```

Comment:
Rather than touching all these sub-directory lit files, we could create a new `lit.local.cfg` file in `sycl/test-e2e/Graph` with `config.unsupported_features += ['gpu-intel-dg2']`.

Reply:
Good idea, done!